### PR TITLE
update opendatasoft documentation URL

### PR DESCRIPTION
**Pull request by vbarbaresi**
# Pull request motivation(s)
Feature request:  Today we changed the default URL from docs.opendatasoft to help.opendatasoft/platform/
We need to perform the indexation on this new URL.

### CODE REVIEW

File: configs/opendatasoft.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
         ]
       }
     },
-    "https://docs.opendatasoft.com/"
+    "https://help.opendatasoft.com/"
```
Comment (s-pace) [inline configs/opendatasoft.json]: This URL is too broad considering your current set of selectors. Please remove it or update your selectors.
Please note that DocSearch is only provided for documentation

Comment (s-pace) [reply configs/opendatasoft.json]: Please provide more details if you want us to encompass more pages

Comment (vbarbaresi) [reply configs/opendatasoft.json]: Right, I removed this URL, the other one is enough.
I have updated the branch.

**Comment by vbarbaresi** [comment]
Thanks!

**Pull request merged**